src/Component/Console/src/DependencyInjection/AddConsoleCommandPass.js

```diff
@@ -0,0 +1,49 @@
+const Command = Jymfony.Component.Console.Command;
+const CompilerPassInterface = Jymfony.Component.DependencyInjection.Compiler.CompilerPassInterface;
+
+/**
+ * @memberOf Jymfony.Component.DependencyInjection.Compiler
+ */
+class AddConsoleCommandPass extends implementationOf(CompilerPassInterface) {
+    /**
+     * @inheritDoc
+     */
+    process(container) {
+        let commandServices = container.findTaggedServiceIds('console.command');
+        let serviceIds = { };
+
+        for (let [ id, tags ] of __jymfony.getEntries(commandServices)) {
+            let definition = container.getDefinition(id);
+            let className = container.parameterBag.resolveValue(definition.getClass());
+
+            let r = new ReflectionClass(className);
+            if (! r) {
+                throw new InvalidArgumentException(
```

Comment:
Unreachable code: ReflectionClass constructor throws if class could not be found.